packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -577,8 +585,9 @@ export class AccountHandler {
         authSalt,
         this.config.verifierVersion
       );
+      const metricsContext = await request.gatherMetricsContext({});
+      await this.signupUtils.verifyAccount(request, account, {});
```

Comment:
I _think_ this is fine. Are we sure we want to do everything in `verifyAccount`? (just double checking)

Reply:
I believe so. We send a post verify email for sync users in this function but that wouldn't really get triggered in flow.